docs/index.md

```diff
@@ -69,6 +67,32 @@ Oidc.Log.logger = console;
 Also, logging has levels so you can control the verbosity by setting the `Oidc.Log.level` to one of `Oidc.Log.NONE`, `Oidc.Log.ERROR`, `Oidc.Log.WARN`, or `Oidc.Log.INFO`. The default is `Oidc.Log.INFO`.
 
 
+## Provider specific settings
+Additional provider specific settings may be needed for a flawless operation:
+
+**Amazon Cognito**
+```
+var mgr = new UserManager({
+    ...
+    revokeTokenTypes = ["refresh_token"]
+});
+```
```

Comment:
this should probably use some valid syntax highlighting

````suggestion
```javascript
const mgr = new UserManager({
    // ...
    revokeTokenTypes: ["refresh_token"]
});
```
````
```